intel_extension_for_transformers/tools/utils.py

```diff
@@ -34,34 +34,10 @@
 def supported_gpus():
     return ['flex', 'max', 'arc']
 
-def get_gpu_family():
-    """Get gpu device family info.
-
-    Return 'flex'|'max'|'arc'| 'no_gpu'| assert
-
-    Note, this function need to import intel_extension_for_pytorch
-
-
-    Additional info (common gpu name):
-      'Intel(R) Data Center GPU Flex 170'
-      'Intel(R) Data Center GPU Max 1100'
-      'Intel(R) Arc(TM) A770 Graphics'
-    """
-
+def is_gpu_available():
```

Comment:
How about is_intel_gpu_available?

Reply:
renamed https://github.com/intel/intel-extension-for-transformers/pull/1479/commits/32670ca3e54c600266cd44c08b34547615b6ea20